### ISSUE TITLE
No exported member 'ModuleBase' with 8.3.0

### ISSUE THREAD

**Issue by joshcomley** [faker-js/faker#2547]
### Pre-Checks

- [X] Follow our [Code of Conduct](https://github.com/faker-js/faker/blob/next/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md).
- [X] Read the [docs](https://fakerjs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/faker-js/faker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Faker issue and not related to a combination with another package.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/faker-js/faker/discussions) or join our [Discord Chat Server](https://chat.fakerjs.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
- [ ] I am willing to provide a PR.

### Describe the bug

Compiling in an Angular 17 project (8.2.0 worked), I get:

```
Module '"../../internal/module-base"' has no exported member 'ModuleBase'.

1 import { ModuleBase } from '../../internal/module-base';
```

Reverting to 8.2.0 fixes it.

### Minimal reproduction code

I will try and get a sample together and update

### Environment Info

```shell
System:
    OS: Windows 11 10.0.22631
    CPU: (16) x64 Intel(R) Core(TM) i7-10875H CPU @ 2.30GHz
    Memory: 26.12 GB / 63.75 GB
  Binaries:
    Node: 18.18.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.15 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 9.8.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Chromium (118.0.2088.76)
    Internet Explorer: 11.0.22621.1
```

### Which module system do you use?

- [ ] CJS
- [ ] ESM

### Used Package Manager

npm

**Comment by ST-DDT** [comment]
I can reproduce the error with the playground project.

**Comment by ST-DDT** [comment]
Preparing release: https://github.com/faker-js/faker/pull/2549

- https://github.com/faker-js/faker/pull/2549

**Comment by joshcomley** [comment]
Can confirm fixed in `8.3.1`
